operationsgateway_api/src/experiments/unique_worker.py

```diff
@@ -0,0 +1,133 @@
+from functools import wraps
+import logging
+import os
+from pathlib import Path
+
+from operationsgateway_api.src.config import Config
+
+
+log = logging.getLogger()
+
+
+class UniqueWorker:
+    """
+    Where multiple workers are launched for this API (by uvicorn for example), this
+    class is used to assign a task to a single worker. For example, the background task
+    to contact the Scheduler on a regular basis only needs to be performed by a single
+    worker. A process ID is to the 'assigned worker' is written to a file and checked
```

Comment:
"A process ID is to the 'assigned worker'" needs a bit of rewording

Reply:
Fixed in c861b49bbf35e931dd8b849155181d38ca7a320a, hopefully it makes more sense now?